### DISCUSSION TITLE
Cpp Syntax

### DISCUSSION THREAD

**Discussion by mjablons1** [mjablons1/concurrency-stopwatch#3] [Q&A]
@Jabkowski Are these class attributes (attached to a class definition rather than to an instance)?

https://github.com/mjablons1/concurrency-stopwatch/blame/6deabe4ca7f2327179eb2ee2e31a841126a6a7c2/cpp_stopwatch/stopwatch.cpp#L52

**Comment by Jabkowski** [comment]
Yes, those are private class attributes. Accessable only in class definition not from instance.
